lib/entities/comment.ts

```diff
@@ -38,9 +38,15 @@ export type RichTextBodyProperty = 'rich-text'
 export type RichTextBodyFormat = { bodyFormat: RichTextBodyProperty }
 export type PlainTextBodyFormat = { bodyFormat?: PlainTextBodyProperty }
 
+export enum CommentStatus {
+  'active',
+  'resolved',
+}
+
 export type CommentProps = {
   sys: CommentSysProps
   body: string
+  status?: CommentStatus
```

Comment:
issue: I think `status` is always available and defaulting to `active`

Reply:
yes! for get, `status` is always there, just or PUT/POST you do not need to provide it.

Reply:
But what about existing comments that have not yet the field set?

Reply:
Then we would need to update these types for those cases:

https://github.com/contentful/contentful-management.js/blob/c80524ca262e18a7ac59c48be310b916691a9a17/lib/entities/comment.ts#L52-L55

Reply:
@Lelith comments that were created before this property existed will return `active`

Reply:
ok, will change it to non optional :)